hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconControllerModule.java

```diff
@@ -136,6 +138,12 @@ protected void configure() {
     }
   }
 
+  @Provides
+  @Singleton
+  public ExecutorService provideReconExecutorService() {
+    return Executors.newFixedThreadPool(5);
```

Comment:
I don't think this should be shared between different tasks, otherwise unrelated tasks may delay each other.  We need a specific pool for each kind of task, each with 2 or 3 threads, depending on the task's parallelization level.  Or we should be using the existing task thread pool (managed in `ReconTaskControllerImpl`), which is configurable.

Also, I guess the threads should be daemons, or the executor should be shutdown somewhere when stopping Recon.